Dockerfile

```diff
@@ -1,3 +1,17 @@
-FROM scratch
+FROM golang:alpine
+
+RUN apk add --no-cache  git
 
 COPY . /go/src/github.com/tmc/pqstream
+
+RUN go get -v github.com/tmc/pqstream/cmd/pqs \
+    && go get -v github.com/tmc/pqstream/cmd/pqsd
+
+RUN go install github.com/tmc/pqstream/cmd/pqs \
+    && go install github.com/tmc/pqstream/cmd/pqsd
+
+ENV PATH $PATH:$GOPATH/bin
+
+EXPOSE 7000
+
+ENTRYPOINT ["pqsd"]
```

Comment:
entrypoint vs CMD implies we wouldn't be able to run the client (pqs) via the same image

Reply:
ah sorry about this - I did remember you mentioned CMD on Slack, but after reading some entrypoint vs cmd docs, I decided to go with entrypoint, b/c it seemed that ti would be possible for users to override the command in CMD - I was prob. confused!